paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -146,17 +146,18 @@ CpuPassStrategy::CpuPassStrategy() : PassStrategy({}) {
                   // "seqpool_concat_fuse_pass",    //
                   "seqpool_cvm_concat_fuse_pass",  //
                   // "embedding_fc_lstm_fuse_pass", //
-                  "fc_lstm_fuse_pass",             //
-                  "mul_lstm_fuse_pass",            //
-                  "fc_gru_fuse_pass",              //
-                  "mul_gru_fuse_pass",             //
-                  "seq_concat_fc_fuse_pass",       //
-                  "fc_fuse_pass",                  //
-                  "repeated_fc_relu_fuse_pass",    //
-                  "squared_mat_sub_fuse_pass",     //
-                  "conv_bn_fuse_pass",             //
-                  "conv_eltwiseadd_bn_fuse_pass",  //
-                  "is_test_pass",                  //
+                  "fc_lstm_fuse_pass",              //
+                  "mul_lstm_fuse_pass",             //
+                  "fc_gru_fuse_pass",               //
+                  "mul_gru_fuse_pass",              //
+                  "seq_concat_fc_fuse_pass",        //
+                  "fc_fuse_pass",                   //
+                  "repeated_fc_relu_fuse_pass",     //
+                  "squared_mat_sub_fuse_pass",      //
+                  "conv_bn_fuse_pass",              //
+                  "conv_eltwiseadd_bn_fuse_pass",   //
+                  "is_test_pass",                   //
+                  "remove_reshape_transpose_pass",  //
```

Comment:
same here https://github.com/PaddlePaddle/Paddle/pull/16342#discussion_r317476841

Reply:
BTW, this PR is doing a big fuse right?
So the name of `remove_reshape_transpose_pass` seems be not fair then, it's just say removing these two ops.

Reply:
This PR is not the big fuse. As described in Brian's document, this pass is served to remove the redundant reshape and transpose ops in the BERT.

Reply:
Did not get the point if do not do some big fuse, then how can just remove transpose and reshape.